instagram_own_version/settings.py

```diff
@@ -49,7 +49,9 @@
 REST_FRAMEWORK = {
     "DEFAULT_AUTHENTICATION_CLASSES": [
         "rest_framework.authentication.TokenAuthentication",
-    ]
+    ],
+    "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.LimitOffsetPagination",
```

Comment:
Pagination is not related to this PR. Please create a separate one

Reply:
Removed